data/pokedex.js

```diff
@@ -11472,7 +11472,7 @@ let BattlePokedex = {
 		types: ["Dragon", "Ground"],
 		gender: "N",
 		baseStats: {hp: 108, atk: 100, def: 121, spa: 81, spd: 95, spe: 95},
-		abilities: {0: "Aura Break", H: "Power Construct"},
+		abilities: {0: "Aura Break", S: "Power Construct"},
```

Comment:
No, Power Construct is definitely Zygarde's Hidden Ability.

Reply:
`S` means "alternate forme's primary ability".

Reply:
Check the mechanics discord, #showdown-coding

Reply:
My mistake, I forgot that Zygarde-50 was Zygarde.